lib/install/management/create.go

```diff
@@ -116,11 +116,11 @@ func (d *Dispatcher) createPool(conf *config.VirtualContainerHostConfigSpec, set
 	return nil
 }
 
-func (d *Dispatcher) uploadImages(files map[string]string) error {
+func (d *Dispatcher) uploadISOs(files map[string]string) error {
 	defer trace.End(trace.Begin("", d.op))
 
 	// upload the images
-	d.op.Info("Uploading images for container")
+	d.op.Info("Uploading vic iso images")
```

Comment:
lower case `vic` - probably some sort of UX violation. Don't know that we need it there anyway. `Uploading ISO images` seems sufficient given this is vic-machine.